### PR TITLE
src/main: add help text for 'service' (if configured)

### PR DESCRIPTION
**Pull request by ejoerns**
Signed-off-by: Enrico Jorns <ejo@pengutronix.de>

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -1706,6 +1712,7 @@ static void cmdline_handler(int argc, char **argv)
 			"  checksum\tUpdate a manifest with checksums (and optionally sign it)\n" \
 			"  install\tInstall a bundle\n" \
 			"  info\t\tShow file information\n" \
+			SERVICE_HELP_TEXT \
```
Comment (jluebbe) [inline src/main.c]: As this is used only in one place, why use the define above (instead of just `#if` / `#endif`)?

Comment (ejoerns) [reply src/main.c]: I had some troubles with the multi-line string and #if in between. If you have a working solution, I would agree that this is preferable ;)

Comment (a3f) [reply src/main.c]: Drop the trailing backslashes and it will work.

Comment (ejoerns) [reply src/main.c]: @a3f You're right. I thought it should work and tested it, but obviously did something wrong or drew the wrong conclusions.. I've now removed them all and inserted the define in-place. Looks much cleaner now :+1:

**Pull request merged**